### DISCUSSION TITLE
Option to disable backtrace prompt

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#17757] [Feature requests] [answered]
I would like to always upload crashes to backtrace.io, but last time my game crashed the window asking yes or no closed itself. If I could always say yes I would

**Comment by janisozaur** [comment]
Sounds doable

**Comment by Broxzier** [comment]
This has to be something players have to set themselves. Having this on by default would be a bad design choice, much like telemetry settings.

**Comment by 733737** [comment]
there is already a debug category in the options menu, there could be a checkbox that defaults to unchecked there that says "upload to backtrace.io automatically"

**Reply by janisozaur** [reply]
It will probably only be available as ini option - or an option when there is a crash prompt

**Reply by 733737** [reply]
ini option is fine for me

**Comment by 733737** [comment]
@janisozaur has this been implemented yet?

**Reply by janisozaur** [reply, marked as answer]
Yes, 8 years ago: https://github.com/OpenRCT2/OpenRCT2/pull/3544

**Reply by 733737** [reply]
do i need to build from source to use it?

**Reply by janisozaur** [reply]
No, just pass that option to executable
